pkg/option/config.go

```diff
@@ -3230,6 +3238,7 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 	c.EnablePMTUDiscovery = vp.GetBool(EnablePMTUDiscovery)
 	c.IPv6NAT46x64CIDR = defaults.IPv6NAT46x64CIDR
 	c.IPAMCiliumNodeUpdateRate = vp.GetDuration(IPAMCiliumNodeUpdateRate)
+	c.EnableMacSpoofCheck = vp.GetBool(EnableMacSpoofCheck)
```

Comment:
Do we need a config flag? Can this be enabled by default instead? Maybe follow the same pattern as SIP verification: add a field to `DatapathConfiguration` called `DisableSmacVerification` that's set to `false` by default.

Reply:
given the nature of this security feature, I think turn it on by default with an option to disable works for me. But tagging @joestringer to weight in on this decision.